src/worker.cc

```diff
@@ -526,6 +536,16 @@ void parse_locations(const rapidjson::Document& doc,
     if (!had_date_time) {
       add_date_to_locations(options, *locations);
     }
+
+    // If any of the locations had search_filter.exclude_closures set to false,
+    // set the corresponding internal costing attribute, so that loki search
+    // does not drop a start/end candidate when an edge is marked as closed due
+    // to live traffic
```

Comment:
NIT:

```suggestion
    // we tell the costing to let all closed roads through so that we can do
    // a secondary per-location filtering using lokis search_filter functionality
```